src/acceptance/assets/app/go_app/deploy.sh

```diff
@@ -1,37 +1,44 @@
 #!/bin/bash
-
 set -euo pipefail
-config=${CONFIG:-}
+
+config="$(cat "${CONFIG:-}")"
+
 function getConfItem(){
-  val=$(jq -r ".$1" "${config}")
+  val=$(jq -r ".$1" <<< "${config}")
```

Comment:
There are two other alternative options:
 + If `${CONFIG}` is set then directly define `readonly config="$(realpath "${CONFIG}")"`.
 + In getConfItem, make use of the root-directory of the git-repo: `val=$(jq -r ".$1" "$(git rev-parse --show-toplevel)/${config}")`.

(I don't say that they are better but perhaps one may be of interest.)

Reply:
What is the problem with storing the complete content in a variable? Reading stuff from files via path is so error prone. I really prefer reading it once to reduce the possible errors as much as possible.

WDYT?

Reply:
Bash has some limitations regarding the variable size. But our config is small enough. For me it would be fine to store the content there.

From the two options I have written above, I like the first one more. With “absolutizing” ( :) ) the path, it should not be error-prone anymore.